tools/link.py

```diff
@@ -749,6 +749,11 @@ def phase_linker_setup(options, linker_args):  # noqa: C901, PLR0912, PLR0915
     if s in user_settings:
       diagnostics.warning('deprecated', f'{s} is deprecated ({reason}). Please open a bug if you have a continuing need for this setting')
 
+  # Set the EXPORT_ES6 default early since it effects the setting of the
```

Comment:
```suggestion
  # Set the EXPORT_ES6 default early since it affects the setting of the
```
affects/effects is truly one of the most bizarre things in English... each has multiple interweaving meanings with the other